### ISSUE TITLE
CMake build installs library in the wrong directory (does not respect LIB_INSTALL_DIR or LIB_SUFFIX)

### ISSUE THREAD

**Issue by thiagomacieira** [google/double-conversion#79]
```
$ cmake -DLIB_INSTALL_DIR:PATH=/usr/lib64 -DLIB_SUFFIX=64 ..
[...]
CMake Warning:
  Manually-specified variables were not used by the project:

    LIB_INSTALL_DIR
    LIB_SUFFIX
$ make
[...]
$ make -s install
[100%] Built target double-conversion
Install the project...
-- Install configuration: ""
CMake Error at cmake_install.cmake:41 (file):
  file cannot create directory: /usr/local/lib.  Maybe need administrative
  privileges.
```

The lib dir `/usr/local/lib` is wrong. It should be `lib64`.

**Comment by thiagomacieira** [comment]
Suggestion: use `include(GNUInstallDirs)`. See https://rix0r.nl/blog/2015/08/13/cmake-guide/

**Comment by Arfrever** [comment]
Fixed recently in #93.

**Comment by floitsch** [comment]
thanks. Closing.
